content/repositories/configuring-branches-and-merges-in-your-repository/managing-rulesets/creating-rulesets-for-a-repository.md

```diff
@@ -14,6 +14,10 @@ shortTitle: Create a ruleset
 
 You can create rulesets to control how users can interact with selected branches and tags in a repository. When you create a ruleset, you can allow certain users to bypass the rules in the ruleset. This can be users with certain permissions, specific teams, or {% data variables.product.prodname_github_apps %}. For more information on rulesets, see "[AUTOTITLE](/repositories/configuring-branches-and-merges-in-your-repository/managing-rulesets/about-rulesets)."
 
+### Quick Start
+
+To get started, GitHub has created [Ruleset Recipes](https://github.com/github/ruleset-recipes) to help get started quick!
+
```

Comment:
```suggestion
You can create rulesets to control how users can interact with selected branches and tags in a repository. When you create a ruleset, you can allow certain users to bypass the rules in the ruleset. This can be users with certain permissions, specific teams, or {% data variables.product.prodname_github_apps %}. For more information on rulesets, see "[AUTOTITLE](/repositories/configuring-branches-and-merges-in-your-repository/managing-rulesets/about-rulesets)."

To import one of  {% data variables.product.prodname_dotcom %}'s pre-made ruleset recipes, see [Ruleset recipes](https://github.com/github/ruleset-recipes). 

```

Reply:
a) you have a double space before the {%
b) same comment regarding repetition, "…made ruleset recipes, see ruleset recipes" :/

Reply:
it might be better acknowledging it's a repo link putting it like:

```suggestion
You can create rulesets to control how users can interact with selected branches and tags in a repository. When you create a ruleset, you can allow certain users to bypass the rules in the ruleset. This can be users with certain permissions, specific teams, or {% data variables.product.prodname_github_apps %}. For more information on rulesets, see "[AUTOTITLE](/repositories/configuring-branches-and-merges-in-your-repository/managing-rulesets/about-rulesets)."

To import one of the pre–made rulesets by {% data variables.product.prodname_dotcom %}, see [`github/ruleset-recipes`](https://github.com/github/ruleset-recipes).
```